docs/build/tools/wallets/kaia-safe/tx-builder.md

```diff
@@ -4,7 +4,38 @@ This is a custom app in Kaia Safe that is responsible for batching transactions.
 
 With transaction builder, you can compose transactions from token transfers to complex contract interactions and batch them into a single transaction.
 
-**Illustration**
+
+## KLAY Token Transfer <a id="token-transfer"></a>
+You can perform token transfer using transaction builder by following the steps below:
+
+**Step 1:** Navigate to Safe Apps and open Transaction Builder Safe App 
+
+![](/img/build/tools/kaia-safe/ks-tx-builder.png)
+
+**Step 2:** Enter the recipient wallet address. For this guide, kindly skip the ABI field as we are trying to execute KLAY transfer transaction.
+
+![](/img/build/tools/kaia-safe/tx-builder-token-recipient-addr.png)
+
+
+**Step 3:** Enter the KLAY value you want to send. 
```

Comment:
can you clearly mention here with a small example, something like if you want to transfer 1 KLAY, give 1 in the input box ?